glade/journal.glade

```diff
@@ -1,261 +1,190 @@
-<?xml version="1.0" standalone="no"?> <!--*- mode: xml -*-->
-<!DOCTYPE glade-interface SYSTEM "http://glade.gnome.org/glade-2.0.dtd">
-
+<?xml version="1.0" encoding="UTF-8"?>
 <glade-interface>
-<requires lib="gnome"/>
-<requires lib="bonobo"/>
-
-<widget class="GnomeApp" id="Journal Window">
-  <property name="visible">True</property>
-  <property name="title" translatable="yes">GnoTime: Journal</property>
-  <property name="type">GTK_WINDOW_TOPLEVEL</property>
-  <property name="window_position">GTK_WIN_POS_NONE</property>
-  <property name="modal">False</property>
-  <property name="default_width">550</property>
-  <property name="default_height">550</property>
-  <property name="resizable">True</property>
-  <property name="destroy_with_parent">False</property>
-  <property name="enable_layout_config">True</property>
-  <signal name="destroy" handler="on_close_clicked"/>
-
-  <child internal-child="dock">
-    <widget class="BonoboDock" id="dock2">
-      <property name="visible">True</property>
-      <property name="allow_floating">True</property>
-
-      <child>
-	<widget class="BonoboDockItem" id="dockitem4">
-	  <property name="border_width">1</property>
-	  <property name="visible">True</property>
-	  <property name="shadow_type">GTK_SHADOW_OUT</property>
-
-	  <child>
-	    <widget class="GtkToolbar" id="toolbar2">
-	      <property name="border_width">1</property>
-	      <property name="visible">True</property>
-	      <property name="orientation">GTK_ORIENTATION_HORIZONTAL</property>
-	      <property name="toolbar_style">GTK_TOOLBAR_BOTH</property>
-	      <property name="tooltips">True</property>
-
-	      <child>
-		<widget class="button" id="publish">
```

Comment:
Glade 3.8 doesn't seem to know what to do with class "button" and resorts to just deleting them. Experimenting with recreating the toolbar buttons outputs class "GtkToolButton" instead.